spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/util/LongUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.cloud.sleuth.util;
+
+
+import org.springframework.util.StringUtils;
+
+/**
+ * Helper class to convert Long to String or return null
+ *
+ * @author Marcin Grzejszczak
+ */
+public class LongUtils {
```

Comment:
I think I mentioned this before, but let's use fixed-length lower-hex encoding.

encoding as a integer will lead to overflows and is unnecessarily different than zipkin for example. It also allows for a negative sign, which can be harder to copy/paste. Moreover, this is used in propagation headers, too, and there's no reason to be different there.

Here's a few places where lower-hex encoding is used.
https://github.com/openzipkin/brave/blob/master/brave-core/src/main/java/com/github/kristofa/brave/IdConversion.java
https://github.com/openzipkin/zipkin-java/blob/master/zipkin-java-core/src/main/java/io/zipkin/internal/JsonCodec.java#L55
https://github.com/twitter/finagle/blob/develop/finagle-core/src/main/scala/com/twitter/finagle/tracing/Id.scala#L20
https://github.com/openzipkin/zipkin/blob/master/zipkin-common/src/main/scala/com/twitter/zipkin/json/JsonSpan.scala#L49